content/docs/user-guide/packaging/project-export/project-export-cli.md

```diff
@@ -17,6 +17,24 @@ To make best use of the `export-project` command, it is recommended to have a pr
 To learn more about the AssetBundler and Seed Files, please visit the [overview page on the AssetBundler tool](https://docs.o3de.org/docs/user-guide/packaging/asset-bundler/overview/).
 
 ## Getting Started
+### TL;DR
+For any new project that you quickly want to test with export, assuming you are using the Default Level supplied with the project as your base, you can quickly do this by double clicking on the [`export.bat`](https://github.com/o3de/o3de/blob/480ce34ab7f3056113e17056125c89ae220bef7a/Templates/DefaultProject/Template/export.bat) script (or running the [`export.sh`](https://github.com/o3de/o3de/blob/480ce34ab7f3056113e17056125c89ae220bef7a/Templates/DefaultProject/Template/export.sh) script on Linux) in the root of your Project folder.
+
+Project Export functionality is also available on a per-project basis. This is because there are helper export scripts that are included in the Project Templates, as can be seen with. These scripts are included at the root folder of every new Project created with the `DefaultProject` and `MinimalProject` templates.
+
+The package scripts pre-populate the relevant arguments for the given project, and then call `export-project` on the user's behalf.
+
+If you need to run it from the command line, simply run the following command (assuming project has already been created and registered):
+```
+# On Windows
+\path\to\project\export.bat
+
+#On Linux
+\path\to\project\export.sh
+```
```

Comment:
For command line examples for different platforms, separate into different code snippets. The '#' comment is for scripts, not command line examples, so that should not be in the snippet, and make it so its something that can be copy/pasted directly into the terminal. You can also use tabs. (note that I had to escape the triple back-tick in the suggestion otherwise the suggestion will be cut-off. Don't apply directly in this PR, but make the change locally with the escaped backticks removed.

```suggestion

{{< tabs name="Export Script" >}}
Assuming `$PROJECT_HOME` is set to the root of the newly created project.
{{% tab name="Windows" %}}
\```shell
$PROJECT_HOME/export.sh
\```

{{% /tab %}}
{{% tab name="Linux" %}}
Assuming `%PROJECT_HOME%` is set to the root of the newly created project.
{{% tab name="Windows" %}}
\```shell
%PROJECT_HOME%/export.bat
\```
{{% /tab %}}
{{< /tabs >}}

```